packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -83,7 +83,7 @@ export default function DocumentActions( { documentTitle } ) {
 			} ) }
 		>
 			{ documentTitle ? (
-				<>
+				<h1>
 					<div
```

Comment:
I think the h1 should only wrap the text of the main item (rather than the whole thing)

Reply:
I was wondering about this as well.  If we only wrap the main item, then only the main Item will show up in a11y navigation.  So if we have 'Singular -> Heading` selected, all the navigation will see is 'Singular'.  While if we wrap both items, it will show 'Singular Header'.  If I try to take a screenshot with voiceover running it closes the window.  But voiceover shows something like:
```
Headings:
1. Singular
```
If we wrap both in the h1, it would be 
```
Headings:
1. Singular Header
```
We could wrap the second item in an h2:
```
Headings:
1. Singular 
2. Header
```

Reply:
But if the sub-label is useful information, it should be displayed there as well.  And marking it as a separate heading seems a bit odd, I do kind of think the middle example (entire wrapper in h1) makes the most sense?

Reply:
🤔 good point...

IMO, if the idea of "h1" is to describe what the html document is about, then I think it should just be around singular. It's pretty obvious that any of these options are confusing by themselves as well. (i.e., it'd be nice if the h1 said "Edit singular template" or something like that)

I feel like the template part name isn't that important to show here. But that's subjective :D

Reply:
Yeah its a bit odd to think about.  Having the document name as a heading definitely makes sense.  The sub-label is a bit more odd since it is not always there and changes white editing the document.  We can leave the sublabel out completely for now?

> (i.e., it'd be nice if the h1 said "Edit singular template" or something like that)

We could make the h1 wrap around both the main label as well as visually hidden text element that says "Edit Template:" or something similar.  So it would end up as:
```
Headings:
1. Edit Template: Singular
```
?

Reply:
Updated with a visually hidden element just before the main documentTitle to say "Edit document".  So voiceover picks it up as:
```
Headings:
1. Edit document: singuler
```

I figure "document" is more precise than "template", since theoretically different types of documents may be displayed here in the future? 🤔

Reply:
I think we should have it say the entity type. There was a PR recently which changed the title of "document" in document settings to "$postType". So imo it makes sense to follow the pattern of using postType for whatever we are primarily editing